src/game/maps/common.ts

```diff
@@ -96,8 +98,26 @@ export class CommonMap {
         }
     }
 
+    private isValidPosition(position: Position): boolean {
+        const pos = Math.sqrt(position.x ** 2 + position.z ** 2)
+        return pos <= MAX_GROUND && position.y >= GROUND_POS.y
+    }
+
     updateGameState() {
         // TODO: 검증로직
+        this.characters.forEach((character) => {
+            if (!this.isValidPosition(character.position)) {
+                console.warn(
+                    `out of map position for character ${character.id}`
+                )
```

Comment:
```suggestion
                console.warn(
                    `out of map position for character ${character.id}`
                )
```

gameLoop 쪽에는 log를 안넣으면 좋을 것 같아요!
pm2 log 에 무한히 많이 찍힐거라..